### PR TITLE
fix gc job

### PR DESCRIPTION
**Pull request by liu4480**
gc job failed to run when it checks there is stale testbed, this is because it always checks if the testbed is in directory `terraform.tfstate.d`, this works when we deploy testbed and destroy testbed upon job finishes. But gc job does not know its original directory, so we store it in /var/lib/jenkins/, the terraform directory should be specified when calling destroy.sh.
Signed-off-by: Bin Liu <biliu@vmware.com>

### CODE REVIEW

**Comment by liu4480** [comment]
> @liu4480 could you update the PR with what was the issue and what is the fix?

thanks for the reminding, will update

File: ci/jenkins/nephe-gc.sh [hunk 1]
```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
-
+set -x
```
Comment (reachjainrahul) [inline ci/jenkins/nephe-gc.sh:2]: can you move this after copyright

File: ci/jenkins/nephe-gc.sh [hunk 2]
```diff
@@ -20,7 +20,8 @@
 
 _usage="Usage: $0 [--goVcPassword <Password for VC>]
   --goVcPassword          Password to the user name for VC.
-  --terraform-dir         directory for terraform."
```
Comment (reachjainrahul) [inline ci/jenkins/nephe-gc.sh:23]: Capital letter, Directory

Comment (liu4480) [reply ci/jenkins/nephe-gc.sh:23]: done

File: ci/jenkins/nephe-gc.sh (hunk 1 above)
Comment (liu4480) [reply ci/jenkins/nephe-gc.sh]: done

**Pull request closed without merging**